src/ansiblelint/utils.py

```diff
@@ -709,12 +709,15 @@ def get_action_tasks(data: AnsibleBaseYAMLObject, file: Lintable) -> List[Any]:
     ]
 
 
-@lru_cache(maxsize=128)
-def parse_yaml_linenumbers(lintable: Lintable) -> AnsibleBaseYAMLObject:
+@lru_cache(maxsize=0)
```

Comment:
Does 0 make the cache unlimited here?

Reply:
Fixed it to None to make it unlimited after reading the docs. To my surprise they used None instead of 0.